### PR TITLE
Initial text for "how do we use" Elemental

### PR DESCRIPTION
**Pull request by agracey**

### CODE REVIEW

File: asciidoc/components/elemental.adoc [hunk 1]
```diff
@@ -9,4 +9,36 @@ ifdef::env-github[]
...
+
+The Elemental Stack consists of a handful of packages on top of SLE Micro for Rancher:
+
+* elemental-operator - Connects to Rancher Manager and handles MachineRegistration and MachineInventory CRDs.
+* elemental-register - Registers machines via machineRegistrations and installs them via elemental-cli.
+* rancher-system-agent - Runs on the installed system and gets instructions ("Plans") from Rancher Manager what to install and run on the system.
+
+== How does SUSE Edge use Elemental?
+
+We use portions of Elemental for managing remote devices where Metal Kubed is not an option (e.g. there is no BMC or the device is behind a NAT gateway).
```
Comment (atanasdinov) [inline asciidoc/components/elemental.adoc:24]: Nit: I haven't seen "Metal Kubed" references before. Isn't it better to use `Metal^3^`?

Also applies to the other occurrence below.

File: asciidoc/components/elemental.adoc [hunk 2]
```diff
@@ -9,4 +9,36 @@ ifdef::env-github[]
...
+We use portions of Elemental for managing remote devices where Metal Kubed is not an option (e.g. there is no BMC or the device is behind a NAT gateway). 
+It allows for an operator to bootstrap their devices in a lab before knowing when or where they will be shipped to. 
+
+We also use it for remote management of nodes' OS in a declarative way (when Metal Kubed is not an option). 
+
+== Best Practices
+
+=== Installation Media
+
+Currently the best way to build a supported Elemental installation disk for SUSE Edge is described in link:/docs/quickstart/elemental[the Elemental quickstart]
```
Comment (atanasdinov) [inline asciidoc/components/elemental.adoc:33]:
```suggestion
Currently the best way to build a supported Elemental installation disk for SUSE Edge is described in link:../quickstart/elemental.adoc[the Elemental quickstart].
```

File: asciidoc/components/elemental.adoc [hunk 3]
```diff
@@ -9,4 +9,36 @@ ifdef::env-github[]
...
+
+=== Labels 
+
+Elemental tracks it's inventory with the MachineInventory CRD and provides a way to select inventory based on labels. This provides a way for users to predefine most (if not all) of their infrastructure needs prior to hardware even being purchased.
```
Comment (atanasdinov) [inline asciidoc/components/elemental.adoc:37]:
```suggestion
Elemental tracks its inventory with the MachineInventory CRD and provides a way to select inventory based on labels. This provides a way for users to predefine most (if not all) of their infrastructure needs prior to hardware even being purchased.
```

**Review by atanasdinov — APPROVED** [review]
Great start overall. Only noticed few small issues. Feel free to merge after those are addressed.

File: asciidoc/components/elemental.adoc (hunk 1 above)
Comment (agracey) [reply asciidoc/components/elemental.adoc:24]: Yeah, it is. I'll change them when I write more this afternoon.

File: asciidoc/components/elemental.adoc (hunk 3 above)
Comment (agracey) [reply asciidoc/components/elemental.adoc]: I swear I know the difference 🤣 I think I'm so used to autocorrect fixing this for me that my muscle memory just types it wrong each time...

**Pull request merged**